tasks/verify_translations.js

```diff
@@ -17,74 +17,30 @@
  * under the License.
  */
 
-// TODO: Integrate a new tool for translations checking
-// https://github.com/elastic/kibana/pull/19826
-import { i18nLoader } from '@kbn/i18n';
+const { resolve } = require('path');
 
-import { toArray } from 'rxjs/operators';
-import { fromRoot, formatListAsProse } from '../src/utils';
-import { findPluginSpecs } from '../src/plugin_discovery';
-import { collectUiExports } from '../src/ui';
-
-import * as i18nVerify from './utils/i18n_verify_keys';
-
-export default function (grunt) {
-  grunt.registerTask('verifyTranslations', async function () {
+module.exports = function (grunt) {
+  grunt.registerTask('verifyTranslations', function () {
```

Comment:
__issue:__ hmm, can you please make `run:i18nCheck` task exactly like `run:typeCheck` (define in `config/run.js`, run it right after `'run:typeCheck'` in `jenkins:unit` and `test` grunt tasks, with `!grunt.option('quick') ` for the latter task)?

And let's rename `scripts/extract_default_translations.js` to `scripts/i18n_check` for now and later on we'll figure out whether to keep this name for extraction purpose or not.